### PR TITLE
Refactor wording

### PR DESCRIPTION
**Pull request by josecelano**
Issue: https://github.com/Nautilus-Cyberneering/git-queue/issues/21

### CODE REVIEW

File: src/stored-message.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 import {nullCommit} from './commit'
 
-export const CREATE_JOB_SUBJECT_PREFIX = 'CLAIM LOCK: JOB: '
-export const MARK_JOB_AS_DONE_SUBJECT_PREFIX = 'RELEASE LOCK: JOB DONE: '
+export const NEW_JOB_SUBJECT_PREFIX = '📝🈺: '
+export const FINISHED_JOB_SUBJECT_PREFIX = '📝✅: '
+
+// TODO: Code Review: should be only use our own class Commit?
+// We are exposing the class Commit in the main module and using DefaultLogFields internally.
+// We can keep doing that but we should rename "nullCommit()" function to "emptyDefaultLogFields()".
```
Comment (yeraydavidrodriguez) [inline src/stored-message.ts:9]: It actually is somehow confusing the conceptual relation between "commit" and "LogFields". Are they the same? if not, what is the difference between a commit and logFields?. Also I don't like at all that last name, is very generic and inexpressive.

File: __tests__/e2e/main.test.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ describe('GitHub Action', () => {
       ...process.env,
       INPUT_QUEUE_NAME: 'QUEUE-NAME',
       INPUT_GIT_REPO_DIR: gitRepoDir,
-      INPUT_ACTION: 'next-job',
+      INPUT_ACTION: 'next-job2',
```
Comment (yeraydavidrodriguez) [inline __tests__/e2e/main.test.ts:55]: Just to confirm: is this '2' right here?

**Review by yeraydavidrodriguez — COMMENTED** [review]
Looks good. Just a minor observation and my opinion on the open question.

**Comment by josecelano** [comment]
hi @yeraydavidrodriguez  @da2ce7 I've just realized that in order to add the job reference (commit hash) on the "finish job" commit subject I need to know (ovbiously) the hash:

```text
📝✅: queue_name: job.ref.1e31b549c630f806961a291b4e3d4a1471f37490
```

I can add an argument to the action command and let the workflow handle that.

Workflow:

```yml
- name: Mark job as finished
  id: finish-job
  if: ${{ steps.create-job.outputs.job_created == 'true' }}
  uses: ./
  with:
    git_repo_dir: ${{ runner.temp }}/temp_git_dir
    queue_name: 'Library Update [library-aaa]'
    action: 'finish-job'
    job_payload: '{"field": "value", "state": "finished"}'
    new_job_commit: "1e31b549c630f806961a291b4e3d4a1471f37490"
```

`src/main.ts`:

```typescript
const markJobAsFinishedCommit = await queue.markJobAsFinished(
  inputs.jobPayload,
  inputs.newJobCommit, /* the commit where the job was created */
  commitOptions
)
```

Notice the new `inputs.newJobCommit` argument.

In the long run, we are going to add job ids, and I think maybe it makes more sense to pass the job id instead of the commit hash. Internally we can search for the commit hash.

If you prefer to use the job id I can finish this PR without adding that part (job.ref.1e31b549c630f806961a291b4e3d4a1471f37490) and create a new issue to add job ids. In fact, I think it's a good idea to add job ids with a single job, before implementing the N-pending-jobs queue.

**Comment by josecelano** [comment]
> hi @yeraydavidrodriguez @da2ce7 I've just realized that in order to add the job reference (commit hash) on the "finish job" commit subject I need to know (ovbiously) the hash:
> 
> ```
> 📝✅: queue_name: job.ref.1e31b549c630f806961a291b4e3d4a1471f37490
> ```
> 
> I can add an argument to the action command and let the workflow handle that.
> 
> Workflow:
> 
> ```yaml
> - name: Mark job as finished
>   id: finish-job
>   if: ${{ steps.create-job.outputs.job_created == 'true' }}
>   uses: ./
>   with:
>     git_repo_dir: ${{ runner.temp }}/temp_git_dir
>     queue_name: 'Library Update [library-aaa]'
>     action: 'finish-job'
>     job_payload: '{"field": "value", "state": "finished"}'
>     new_job_commit: "1e31b549c630f806961a291b4e3d4a1471f37490"
> ```
> 
> `src/main.ts`:
> 
> ```ts
> const markJobAsFinishedCommit = await queue.markJobAsFinished(
>   inputs.jobPayload,
>   inputs.newJobCommit, /* the commit where the job was created */
>   commitOptions
> )
> ```
> 
> Notice the new `inputs.newJobCommit` argument.
> 
> In the long run, we are going to add job ids, and I think maybe it makes more sense to pass the job id instead of the commit hash. Internally we can search for the commit hash.
> 
> If you prefer to use the job id I can finish this PR without adding that part (job.ref.1e31b549c630f806961a291b4e3d4a1471f37490) and create a new issue to add job ids. In fact, I think it's a good idea to add job ids with a single job, before implementing the N-pending-jobs queue.

Ignore my previous comment. With my fresh brain in the morning, I see I can get the commit because there is only one pending job. I was already thinking about multiple jobs. In fact, before adding the "finish" commit I check that there is a pending job and I have the commit hash in the StoredMessage.

Anyway, I think it's a good idea the split the [multi-job issue](https://github.com/Nautilus-Cyberneering/git-queue/issues/30) into 2 issues. We can implement first an issue to add the job number and then allow multiple pending jobs.

**Comment by josecelano** [comment]
The feature is finished but I want to refactor some parts of the code (minor changes, extract some classes to move validation to them).

**Comment by josecelano** [comment]
hi @yeraydavidrodriguez @da2ce7 this is finished.

**Pull request merged**